web/app/js/components/LineGraph.jsx

```diff
@@ -133,6 +135,40 @@ export default class LineGraph extends React.Component {
     }
   }
 
+  flashLatestDataPoint() {
+    this.svg.select("circle").remove();
+    let circleData = {};
+
+    if (!_.isEmpty(this.props.data) && this.props.showFlash) {
+      circleData = this.props.data.slice(-1)[0];
+      this.svg.append("circle")
+        .attr("class", "flash")
+        .style("opacity", 0.6)
+        .attr("cx", () => {
+          if (circleData.timestamp) {
+            return this.xScale(circleData.timestamp);
+          }
+        })
+        .attr("cy", () => {
+          if (circleData.value) {
+            return this.yScale(circleData.value);
+          }
+        })
+        .attr("r", 6)
+        .transition()
+        .on("start", function repeat() {
+          d3.active(this)
+            .transition()
+            .duration(550)
+            .style("opacity", 0.6)
+            .transition()
+            .duration(550)
+            .style("opacity", 0)
+            .on("start", repeat);
```

Comment:
Is this `repeat` redundant given that we're calling `flashLatestDataPoint()` on every `updateGraph()` Assuming so, I guess we have a couple options:
1. Flash the circle on every `updateGraph()`. This makes the flashing reflect real data updates.
2. Flash the circle on a given interval. This makes the flashing more aesthetically pleasing, as we can set the interval to something that looks good rather than how often data updates.

I sort of lean towards option 2, but I realize the original requirement was to flash on data update. What do you think?

Reply:
yeah I agree that flashing on a given interval is more aesthetically pleasing, and would support that. now that we're updating data every 10s, seems a bit slow. 
I think the flashing was intended to give a feeling of liveness to the page, anyways.

Reply:
Yes, I chatted with @klingerf about this - different graphs update in different intervals, so flashing on `updateGraph()` will be 2 seconds on the servicemesh overview page, but 10 seconds on the deployments page. The inconsistency seemed off.
 
I will experiment flashing on a given interval, though it always resets when `updateGraph()` is called again, so when I started with intervals and delays in d3, it resulted in double flashes every once in a while. I probably just need to experiment with it some more :)

Reply:
I started playing with this a bit! Perhaps there's a way to not add/remove the circle on `updateGraph()`, but rather just shift its position? This way the flashing will be seamless across data updates.